### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by aljones15**
This is needed for veres-wallet.

Think we're ready for a release here?

### CODE REVIEW

File: schemas/account.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+const {schemas} = require('bedrock-validation');
```
Comment (mattcollier) [inline schemas/account.js]: Please add copyright notice and `'use strict';`

File: test/helpers.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node: true */
```
Comment (mattcollier) [inline test/helpers.js]: remove.

File: test/mock.data.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+'use strict';
```
Comment (mattcollier) [inline test/mock.data.js]: copyright notice.

File: test/package.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    "bedrock-key": "^5.1.2",
+    "bedrock-mongodb": "^5.0.0",
+    "bedrock-passport": "^4.0.4",
+    "bedrock-permission": "^2.4.2",
+    "bedrock-rest": "^2.0.0",
+    "bedrock-server": "^2.1.3",
+    "bedrock-test": "^2.0.0",
+    "bedrock-validation": "^4.0.0",
+    "bedrock-views": "^5.4.1",
+    "eslint-config-digitalbazaar": "^1.1.0",
```
Comment (mattcollier) [inline test/package.json]: this is not required here is it?  It should be getting installed at the root level?

**Comment by aljones15** [comment]
in the future probably not a good idea to have PR comments on branches you can not push to. dev is locked also. creating a new branch for these comments.

**Pull request closed without merging**